preload_data.sh

```diff
@@ -279,6 +280,41 @@ function pre_req_bpm() {
 } # parse
 
 
+#
+# Add full DNS name into icp-mongodb-client-cert certificate
+#
+function patch_cert() {
+    info "Adding full DNS name into icp-mongodb-client-cert certificate in $FROM_NAMESPACE"
+
+    full_dnsname=$(${OC} get certificate icp-mongodb-client-cert -n $FROM_NAMESPACE -o=jsonpath='{.spec.dnsNames}') 
```

Comment:
Can we use full qualified kind name on `certificate` to indicate it is v1 version of `certificate`
```
oc get certificates.v1.cert-manager.io -A
```